Mathlib/Topology/UniformSpace/UniformConvergence.lean

```diff
@@ -137,6 +137,10 @@ theorem tendstoUniformlyOn_iff_tendstoUniformly_comp_coe :
     TendstoUniformlyOn F f p s ↔ TendstoUniformly (fun i (x : s) => F i x) (f ∘ (↑)) p :=
   forall₂_congr fun u _ => by simp
 
+lemma tendstouniformlyOn_iff_restrict {K : Set α} : TendstoUniformlyOn F f p K ↔
```

Comment:
```suggestion
lemma tendstoUniformlyOn_iff_restrict {K : Set α} : TendstoUniformlyOn F f p K ↔
```

I'm surprised we don't have this. Does `apply?` give anything?

Reply:
unfortunately not :(